scripts/ci/install-olm-local

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+TMP_OLM="$(mktemp -d)"
+git clone https://github.com/operator-framework/operator-lifecycle-manager.git "$TMP_OLM"
```

Comment:
I'm currently having an issue with `make run-local` on my own fork using these tests. OLM's upstream values point to an image not present in quay: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/local-values.yaml#L11
There has been conversation about a workaround for this. @alecmerdler